fe/fe-core/src/main/java/com/starrocks/planner/OlapScanNode.java

```diff
@@ -790,6 +790,10 @@ protected void toThrift(TPlanNode msg) {
             if (!bucketExprs.isEmpty()) {
                 msg.lake_scan_node.setBucket_exprs(Expr.treesToThrift(bucketExprs));
             }
+
+            if (CollectionUtils.isNotEmpty(columnAccessPaths)) {
+                msg.lake_scan_node.setColumn_access_paths(columnAccessPathToThrift());
+            }
```

Comment:
there will be some expr cann't be pushed down to lake scan? for example `map[key]` on flat_map?

Reply:
Can support, because I don't know what you need so I don't implement, we can discuss it

Reply:
I'm not sure, but can we send it on first?